### PR TITLE
Admin: Add Event nav item for Legal Documents

### PR DESCRIPTION
**Pull request by sarthaktexas**
## Summary of the problem
Makes a more efficient workflow for admins.

## Describe your changes
Releases a new feature flag `:admin_legal_documents_2024_02_05` for a feature (that allows them to go directly to legal documents) so that admins that don't need this feature don't get it.
image

### CODE REVIEW

File: app/views/events/_nav.html.erb [hunk 1]
```diff
@@ -104,6 +104,15 @@
         tooltip: "Commonly asked questions and how-tos",
         icon: "info",
         selected: local_assigns[:selected] == :documentation %>
+      <% if Flipper.enabled?(:admin_legal_documents_2024_02_05, current_user) %>
+        <% admin_tools("m0 p0 overflow-visible w-fit") do %>
+          <%= dock_item "Legal Documents",
+            event_documents_path(@event),
+            tooltip: "Go directly to legal documents",
```
Comment (sampoder) [inline app/views/events/_nav.html.erb]:
```suggestion
            tooltip: "Fiscal sponsorship contract etc.",
```

Comment (sarthaktexas) [reply app/views/events/_nav.html.erb]: what

Comment (sampoder) [reply app/views/events/_nav.html.erb]: well it felt kind of redundant to repeat legal documents so i thought maybe clarifying what sort of documents there are would be nice

Comment (sarthaktexas) [reply app/views/events/_nav.html.erb]: oh! i see what you mean ✅

**Comment by garyhtou** [comment]
Notes from when @sarthaktexas and I chatted a few weeks ago:
- Let's make this available when right-clicking the existing nav item
- Similar design to right-clicking the HCB icon

**Pull request closed without merging**